README.adoc

```diff
@@ -93,3 +93,19 @@ make sure the installer actually downloads and installs the new DFX release the
    (the CD script) and execute it.
 
 
+== Troubleshooting
+This section provides solutions to problems you might encounter when using `dfx`
+
+=== Project Reset
+
+This command will remove the build directory and restart your replica
+```shell
+dfx stop && dfx start --clean --background
+```
+
+=== Using Internet Identity Locally
+If you're encountering issues using II locally, this command will deploy the Internet Identity canister into your replica.
+
+```shell
+II_ENV=development dfx deploy --no-wallet --argument '(null)'
+```
```

Comment:
Without more context, this section might give someone the idea that they can deploy Internet Identity into their project just by running this command.

The command by itself won't install the II, unless you run it from a clone of https://github.com/dfinity/internet-identity .  And even then, deploying II into one project by running this command from another working directory is a fairly advanced operation.

The command itself is also available here: https://github.com/dfinity/internet-identity#running-locally .

I'm not sure how to word this here, but here is an attempt:
  
```suggestion
=== Using Internet Identity Locally
You can deploy the Internet Identity canister into your replica alongside your project by cloning https://github.com/dfinity/internet-identity, and then from that directory, running the following command:

[source, bash]
----
II_ENV=development dfx deploy --no-wallet --argument '(null)'
----

There are more notes at https://github.com/dfinity/internet-identity#running-locally that may be helpful.
```

Reply:
I updated the wording here. You make a good point